src/Parsing/Test/Microsoft.Python.Parsing.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Is this needed? Does the build need to be bumped for this?

Reply:
Yes, VS gives warning that 30 is no longer supported. LS itself has been 31 for a while, so this is basically forgotten change.